commit 5ad4e41e693e976a49d61bada0d690075e1f2681
Author: Dan Terpstra <terpstra@eecs.utk.edu>
Date:   Sun Sep 9 16:57:42 2012 -0400

    Remove doubly defined events from the combined Sandy / Ivy Bridge event table to fix my previous commit.

diff --git a/src/papi_events.csv b/src/papi_events.csv
index 36630f6..27b6ba9 100644
--- a/src/papi_events.csv
+++ b/src/papi_events.csv
@@ -565,8 +565,6 @@ PRESET,PAPI_L3_TCW,NOT_DERIVED,L2_RQSTS:RFO_MISS
 PRESET,PAPI_BR_TKN,DERIVED_SUB,BR_INST_RETIRED:ALL_BRANCHES,BR_INST_RETIRED:NOT_TAKEN
 PRESET,PAPI_BR_NTK,NOT_DERIVED,BR_INST_RETIRED:NOT_TAKEN
 PRESET,PAPI_BR_INS,NOT_DERIVED,BR_INST_RETIRED:ALL_BRANCHES
-PRESET,PAPI_BR_CN,NOT_DERIVED,BR_INST_RETIRED:CONDITIONAL
-PRESET,PAPI_BR_UCN,DERIVED_SUB,BR_INST_RETIRED:ALL_BRANCHES,BR_INST_RETIRED:CONDITIONAL
 PRESET,PAPI_BR_MSP,NOT_DERIVED,BR_MISP_RETIRED:ALL_BRANCHES
 PRESET,PAPI_BR_PRC,DERIVED_SUB,BR_INST_RETIRED:ALL_BRANCHES,BR_MISP_RETIRED:ALL_BRANCHES
 #

commit 7bcb741fa6797efd8c2932e9dba44fcc90c50124
Author: Dan Terpstra <terpstra@eecs.utk.edu>
Date:   Thu Sep 6 16:50:15 2012 -0400

    Fix event name issue for Ivy Bridge that surfaced in a recent libpfm4 commit.

diff --git a/src/papi_events.csv b/src/papi_events.csv
index c42f693..36630f6 100644
--- a/src/papi_events.csv
+++ b/src/papi_events.csv
@@ -583,6 +583,8 @@ CPU,snb_ep
 #
 PRESET,PAPI_L2_TCW,NOT_DERIVED,L2_RQSTS:RFO_ANY
 PRESET,PAPI_L2_DCH,DERIVED_ADD,L2_RQSTS:ALL_DEMAND_RD_HIT,L2_RQSTS:RFO_HITS
+PRESET,PAPI_BR_CN,NOT_DERIVED,BR_INST_RETIRED:CONDITIONAL
+PRESET,PAPI_BR_UCN,DERIVED_SUB,BR_INST_RETIRED:ALL_BRANCHES,BR_INST_RETIRED:CONDITIONAL
 PRESET,PAPI_TLB_DM,DERIVED_ADD,DTLB_LOAD_MISSES:CAUSES_A_WALK,DTLB_STORE_MISSES:CAUSES_A_WALK
 #
 # Counts scalars only; no SSE or AVX is counted; includes speculative
@@ -598,7 +600,9 @@ PRESET,PAPI_VEC_DP,DERIVED_POSTFIX,N0|2|*|N1|4|*|+|,FP_COMP_OPS_EXE:SSE_FP_PACKE
 CPU,ivb
 #
 PRESET,PAPI_L2_TCW,NOT_DERIVED,L2_RQSTS:ALL_RFO
-PRESET,PAPI_L2_DCH,DERIVED_ADD,L2_RQSTS:DEMAND_DATA_RD_HIT,L2_RQSTS:RFO_HITS
+PRESET,PAPI_L2_DCH,DERIVED_ADD,L2_RQSTS:DEMAND_DATA_RD_HIT,L2_RQSTS:RFO_HIT
+PRESET,PAPI_BR_CN,NOT_DERIVED,BR_INST_RETIRED:COND
+PRESET,PAPI_BR_UCN,DERIVED_SUB,BR_INST_RETIRED:ALL_BRANCHES,BR_INST_RETIRED:COND
 PRESET,PAPI_TLB_DM,DERIVED_ADD,DTLB_LOAD_MISSES:DEMAND_LD_MISS_CAUSES_A_WALK,DTLB_STORE_MISSES:CAUSES_A_WALK
 #
 CPU,Intel Core2
